src/test/java/automation/utils/loaders/GroupPropLoader.java

```diff
@@ -0,0 +1,19 @@
+package automation.utils.loaders;
```

Comment:
Use any of approaches listed here to provide input values into tests: https://dzone.com/articles/alternative-junit.

Reply:
I use different - @ParametrizedTest from JUnit5, but I think it is similar to the approach you want me to use